backend/app/tests/conftest.py

```
@@ -1,17 +1,45 @@
+    get_api_key_by_email,
+    load_environment,
+)
+from app.seed_data.seed_data import seed_database
+
+
+@pytest.fixture(scope="session", autouse=True)
+def load_test_env():
+    load_environment("../.env.test")
+
+
+from app.core.config import settings
```

Ruff (0.12.2)

Module level import not at top of file

(E402)